scanpy/preprocessing/_normalization.py

```diff
@@ -207,7 +225,7 @@ def normalize_total(
     counts_per_cell = np.ravel(counts_per_cell)
 
     cell_subset = counts_per_cell > 0
-    if not np.all(cell_subset):
+    if not isinstance(cell_subset, DaskArray) and not np.all(cell_subset):
```

Comment:
I think it's fine to leave this in so that normalization without `exclude_highly_expressed` just works